pkg/scheduler/scheduler.go

```diff
@@ -175,8 +171,8 @@ func reportSkippedPreemptions(p map[string]int) {
 }
 
 func (s *Scheduler) schedule(ctx context.Context) wait.SpeedSignal {
-	s.attemptCount++
-	log := ctrl.LoggerFrom(ctx).WithValues("attemptCount", s.attemptCount)
+	s.schedulingCycleCount++
+	log := ctrl.LoggerFrom(ctx).WithValues("schedulingCycleCount", s.schedulingCycleCount)
```

Comment:
perhaps more simply `schedulingCycle`? @mimowo

Reply:
+1, I'm leaning to just `schedulingCycle`